projects/aca-content/assets/i18n/en.json

```diff
@@ -211,12 +211,15 @@
               "RESET_ACTION": "Reset search filters",
               "SAVE_SEARCH": {
                 "ACTION_BUTTON": "Save Search",
+                "SAVE_CHANGES": "Save changes",
+                "SAVE_AS_NEW": "Save as new",
                 "MODAL_HEADER": "Save this search",
                 "NAME_LABEL": "Name",
                 "NAME_REQUIRED_ERROR": "This field is required",
                 "DESCRIPTION_LABEL": "Description",
                 "SAVE_SUCCESS": "Search Saved",
                 "SAVE_ERROR": "Error occurred. Search could not be saved.",
+                "SEARCH_NAME_NOT_UNIQUE_ERROR": "Saved Search with {{ name }} name already exists.",
```

Comment:
it might be worth enclosing the `{{ name }}` in single parentheses

Reply:
Why? This is not a number and I think we don't enclose names in parentheses in other cases

Reply:
I can see somewhere we do and in other places we don't, so it is up to you. 
![Screenshot 2024-11-07 at 13 58 22](https://github.com/user-attachments/assets/564811e2-0ad9-47f6-9785-45827820683a)